homeassistant/components/rollershutter/demo.py

```diff
@@ -60,6 +60,10 @@ def move_down(self, **kwargs):
         self._listen()
         self._moving_up = False
 
+    def move_position(self, position, **kwargs):
```

Comment:
You will need to implement it for the demo because otherwise you are unable to write tests for it.